atgu/atgu/templates/resource.html

```diff
@@ -0,0 +1,62 @@
+{% extends "layout.html" %}
+{% block title %}{{ resource['title'] }}{% endblock %}
+{% block head %}
+  <link href="https://cdn.quilljs.com/1.3.6/quill.snow.css" rel="stylesheet" crossorigin="anonymous">
+{% endblock %}
+{% block breadcrumb_items %}
+  <li class="breadcrumb-item"><a href="{{ base_path }}">ATGU</a></li>
+  <li class="breadcrumb-item"><a href="{{ base_path }}">Resources</a></li>
+  <li class="breadcrumb-item active" aria-current="page">{{ resource['title'] }}</li>
+{% endblock %}
+{% block container_contents %}
+  <div class="row">
+    <div class="col">
+      <div style="float:right;">
+        <form action="{{ base_path }}/resources/{{ resource['id'] }}/edit" class="pr-3" style="display:inline;">
+          <button type="submit" class="btn btn-primary">Edit</button>
+        </form>
+
+        <form action="{{ base_path }}/resources/{{ resource['id'] }}/delete" method="post" style="display:inline;">
```

Comment:
This is fine for now, but it would be nice to label what is tags, what is content, what is description, etc.